lib/constants.js

```diff
@@ -36,10 +36,18 @@ const ATTRIBUTE_DEFAULT = ' ';
  * @return     {String}              A default value to use in the entity
  */
 function getInitialValueForType(type) {
-    switch (type) {
-        case LOCATION_TYPE:
+    switch (type.toLowerCase()) {
+        case 'geoproperty':
             return LOCATION_DEFAULT;
-        case DATETIME_TYPE:
```

Comment:
Is there any other usage of LOCATION_TYPE and DATETIME_TYPE (removed in this PR) appart of the ones in this point?

Reply:
Doesn't look like it, so those statements are now redundant and can be removed. Fixed 9d65c09